bench/src/cli.rs

```diff
@@ -1,26 +1,43 @@
+use crate::{helpers::USER_KEYPAIR_PATH, strategies::Strategies, tx_size::TxSize};
 use clap::{command, Parser};
 
 #[derive(Parser, Debug)]
 #[command(author, version, about, long_about = None)]
 pub struct Args {
-    /// Number of tx(s) sent in each run
-    #[arg(short = 'n', long, default_value_t = 5_000)]
-    pub tx_count: usize,
-    /// Number of bench runs
-    #[arg(short = 'r', long, default_value_t = 1)]
-    pub runs: usize,
-    /// Interval between each bench run (ms)
-    #[arg(short = 'i', long, default_value_t = 1000)]
-    pub run_interval_ms: u64,
     /// Metrics output file name
     #[arg(short = 'm', long, default_value_t = String::from("metrics.csv"))]
     pub metrics_file_name: String,
-    /// Lite Rpc Address
-    #[arg(short = 'l', long, default_value_t = String::from("http://127.0.0.1:8890"))]
+    /// strategy for sending transactions
+    #[command(subcommand)]
+    pub strategy: Strategies,
+}
+
+#[derive(clap::Args, Debug)]
+pub struct RpcArgs {
+    /// RPC endpoint
+    #[arg(short = 'r', long, default_value_t = String::from("http://0.0.0.0:8899"))]
+    pub rpc_addr: String,
```

Comment:
naming of `rpc-addr` and `lite-rpc-addr` is quite misleading.  could we do better?